app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -220,4 +220,15 @@ def vm_reconfigure(vm, options = {})
     end
     _log.info("#{log_header} Completed.")
   end
+
+  # Calculates an "ems_ref" from the "href" attribute provided by the oVirt REST API, removing the
+  # "/ovirt-engine/" prefix, as for historic reasons the "ems_ref" stored in the database doens't
+  # contain it, it only contains the "/api" prefix which was used by older versions of the engine.
+  def self.make_ems_ref(href)
+    if href.nil?
+      return nil
```

Comment:
Minor but inline this change... `return nil if href.nil?`

Alternately, just simplifiy this to whole method to a one liner with...

```ruby
href && href.sub(%r{^/ovirt-engine/}, '/')
```